website/docs/docs/cloud/secure/snowflake-privatelink.md

```diff
@@ -97,27 +97,26 @@ Once dbt Cloud support completes the configuration, you can start creating new c
 4. Configure the remaining data platform details.
 5. Test your connection and save it.
 
-## Enable the connection in Snowflake
+## Enable the connection in Snowflake (Azure)
```

Comment:
Parenthesis in headers can be wonky for links. Adding "Azure" to the header instead
```suggestion
### Enable the connection in Snowflake hosted on Azure
```